src/pages/accounts/ForgotPassword.tsx

```diff
@@ -48,13 +55,46 @@ const TextLink = styled(TextButton)`
 	word-wrap: break-word;
 `;
 
+interface InputData {
+	value: string;
+	error: boolean;
+	errorMessage: string;
+}
+
+function inputReducer(state: InputData, action: Partial<InputData>) {
+	return { ...state, ...action };
+}
+
+const initialInputValue: InputData = {
+	value: '',
+	error: false,
+	errorMessage: '',
+};
+
 function validEmail(email: string) {
 	return !!email.match(/.+@.+\..{2,}/);
 }
 
+function validPass(password: string) {
+	if (!password.match(/.{8,}/)) return false; // 8 Characters
+	if (!password.match(/[A-Z]/)) return false; // Upper-Case
+	if (!password.match(/[a-z]/)) return false; // Lower-Case
+	return true; // Passed all checks
```

Comment:
You could replace this with `return password.match(/.{8,}/) && password.match(/[A-Z]/) && password.match(/[a-z]/)`, but that's up to personal preference :]